### PR TITLE
Make probing optional

### PR DESCRIPTION
**Pull request by proflayton**
I would like to add the ability into Sirvice to not airlock. This requires a change here to make it optional

### CODE REVIEW

File: test/typed-request-client.js [hunk 1]
```diff
@@ -529,6 +529,155 @@ test('can make request without filtering response', function t(assert) {
...
+    var treq = {
+        url: 'http://localhost:8000/',
+        method: 'GET',
+        headers: {},
+        body: {
+            'foo': 'bar'
+        }
+    };
+
+    console.log(request);
```
Comment (Raynos) [inline test/typed-request-client.js]: console.log in tests :(

**Comment by Raynos** [comment]
Seems reasonable to me.

**Comment by proflayton** [comment]
Thanks for catching the console.log :/

**Comment by proflayton** [comment]
If this is good, can I get a merge + version bump?

**Comment by Raynos** [comment]
@proflayton you can merge and bump yourself :)

Recommend: minor version

**Comment by proflayton** [comment]
:+1: Didn't know the ownership rules here

**Pull request merged**